frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -36,6 +37,14 @@ export const Root = styled.div<{
       border-radius: 8px;
       box-shadow: 0 2px 2px var(--mb-color-shadow);
     `}
+
+  ${props =>
+    props.hasVisibleOverflowWhenPriting &&
```

Comment:
Is there anything preventing us from not always having this `@media print` block?

Having to worry about one less condition is always better.

Reply:
I believe it might bring back #40660 when printing a page with embedded dashboard

Reply:
can you test that just to make sure our assumption was right? I'm happy to have this prop if needed, but let's add a comment about that.

Reply:
@WiNloSt, I checked it, and if we remove the "public dashboard" check, then things will look broken when printing an embedded dashboard. Here's an example, notice how if we scroll the dashboard for a bit, then its contents take over the entire print preview because of `overflow: visible`:

<details>
<summary>Video</summary>

https://github.com/user-attachments/assets/5101d51f-8551-47ae-800b-3631bf87b16e

</details>

OTOH, here's how things look with the "public dashboard" check. The printing preview only shows the visible part of the dashboard:

<details>
<summary>Video</summary>

https://github.com/user-attachments/assets/c71df1cf-ce6b-4cb3-a6d9-9efb21f5fd06

</details>

I left a comment explaining this right next to the condition